packages/es-client/indexer.js

```diff
@@ -90,7 +90,7 @@ async function genericRecordUpdate(esClient, id, doc, index, type, parent) {
     try {
       indexResponse = await actualEsClient.client.index(params);
     } catch (error) {
-      if (error.name === 'ResponseError' && error.meta.body.message.includes('The security token included in the request is expired')) {
+      if (error.name === 'ResponseError' && error?.meta?.body?.message.includes('The security token included in the request is expired')) {
```

Comment:
Were we hitting an error here? Agreed that it's best practice either way, just curious if we need to update a test or something.

Reply:
Yes - specifically there is an error thrown via the ES call path if I overloaded the local ES host with too many concurrent calls in a unit test 😓  that didn't have error.meta, and then bailed with that error rather than the thrown error. 

I don't think we need to update a test, we just need to not assume anything about the error object in conditionals like these.

Reply:
Yup, sounds great. Agreed not testing required as this really should be what we're doing when we access most (all?) object properties.